src/backend/.gitignore

```diff
@@ -1,2 +1,3 @@
 **/env/
-**/__pycache__/
\ No newline at end of file
+**/__pycache__/
```

Comment:
```
**/**.pyc
tests/**.pyc
.pytest_cache
__init__.pyc
```

Add that to this file and then we don't have to have it in any of the service-level gitingores